src/test/java/com/knoldus/SpringWebFluxApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.knoldus;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class SpringWebFluxApplicationTests {
```

Comment:
What are we testing as part of this test case?

Reply:
i have written the seperate class for the testing purpose